lpc82x/.travis.yml

```diff
@@ -1,17 +0,0 @@
-language: rust
-
-rust:
-  - stable
-  - beta
-  - nightly
-
-addons:
-  apt:
-    packages:
-      - gcc-arm-none-eabi
-
-install:
-  - rustup target add thumbv6m-none-eabi
```

Comment:
Is this included by default? Or just not necessary?

Reply:
It's not included by default, but I don't think it's necessary.

I think it makes no difference whether we compile for the actual target or the host platform. Of course we couldn't run it on the host platform, but that doesn't make a difference for the CI.

I could be mistaken, and compiling for the target might be the safer choice. I opted to keep it simple, in order to keep compile times down. I'm happy to chance this, if you object.

Reply:
Nah I was just curious. It may make a small difference in some cases because of architecture differences but in general I agree with you.